lib/kennel/api.rb

```diff
@@ -23,33 +23,42 @@ def list(api_resource, params = {})
         all = []
 
         loop do
-          result = request :get, "/api/v1/#{api_resource}", params: params.merge(limit: limit, offset: offset)
+          result = request :get, api_url(api_resource), params: params.merge(limit: limit, offset: offset)
           data = result.fetch(:data)
           all.concat data
           break all if data.size < limit
           offset += limit
         end
       else
-        result = request :get, "/api/v1/#{api_resource}", params: params
+        result = request :get, api_url(api_resource), params: params
         result = result.fetch(:dashboards) if api_resource == "dashboard"
         result
       end
     end
 
     def create(api_resource, attributes)
-      reply = request :post, "/api/v1/#{api_resource}", body: attributes
-      api_resource == "slo" ? reply[:data].first : reply
+      attributes = { data: attributes } if api_resource == 'logs/config/metrics'
+      reply = request :post, api_url(api_resource), body: attributes
+      if api_resource == "slo"
+        reply[:data].first
+      elsif api_resource == 'logs/config/metrics'
+        reply[:data]
```

Comment:
if we make the model generate `data: attributes` then we don't need to pack/unpack here
... but not much different either way

Reply:
i tried that but then it shows a permadiff when fetching for updates, since i think it already checks within :data for list?